shell/app/common/components/render-form.tsx

```diff
@@ -80,14 +80,24 @@ class RenderPureForm extends React.Component<IProps> {
     return onlyItems ? (
       items
     ) : (
-      <Form className={formClass} layout={layout} style={style}>
+      <Form form={form} className={formClass} layout={layout} style={style}>
         {items}
       </Form>
     );
   }
 }
 
-const RenderForm = Form.create()(RenderPureForm) as any;
+const RenderForm = forwardRef((props: any, ref) => {
+  const [form] = Form.useForm();
+
+  useImperativeHandle(ref, () => ({
```

Comment:
I think there is no need to wrap a `props`,then we can shorten the call chain
`formRef.current.props.form.validateFields()`  => `formRef.current.validateFields()`